private/configurations/settings/network.ps1

```diff
@@ -0,0 +1,3 @@
+Set-DbatoolsConfig -FullName network.proxy.url -Value $null -Initialize -Validation string -Description "The URL of the network proxy."
+Set-DbatoolsConfig -FullName network.proxy.username -Value $null -Initialize -Validation string -Description "The username for the network proxy."
+Set-DbatoolsConfig -FullName network.proxy.securepassword -Value $null -Initialize -Validation securestring -Description "The secure password for the network proxy."
```

Comment:
Not sure I've ever tried but is there any reason we couldn't just squash these two into a config that holds the credential object itself?

Reply:
@wsmelton i believe this is a limitation of the securestring type. fred said it was very complicated.